tor2web/utils/lists.py

```diff
@@ -258,7 +267,4 @@ def handleData(self, data):
             self.clear()
 
         data = json.loads(data)
-        for relay in data['relays']:
-            for ip in relay['a']:
-                if ip != '':
-                    self.add(ip)
+        self.extend( [ ip for ip in relay['a'] for relay in data['relays'] if ip ] )
```

Comment:
I understand the bottom one is confusing.  However, I would like to keep the rest, especially the .extend() because it mirrors the python list() datatype's .extend() function.